### ISSUE TITLE
release checkpoint or random seeds？

### ISSUE THREAD

**Issue by sakur-a-code** [zxh20001117/AHVE-CNER#1]
I have been closely following the configurations provided in your paper and code. However, the model I have trained myself is consistently achieving results that are significantly lower across various metrics compared to those reported in your paper.
To help me troubleshoot the potential source of this discrepancy, I was wondering if you would be able to share the pre-trained model weights (checkpoints) or the random seeds used in your experiments. Any assistance you could provide would be immensely helpful.

**Comment by zxh20001117** [comment]
i wonder if you are training on the weibo dataset? I came across a issue that the datasets downloaded online differs a lot and some would lead to very bad results (0.60 or so on overall f1 score). And you could try a bigger batch size. In addition, I cancelled the random seed setting in the later training of the experiment.

**Comment by sakur-a-code** [comment]
Thanks for your reply! I was testing on the Weibo dataset. Would you mind sharing the version you used?

**Comment by sakur-a-code** [comment]
如果可以的话能请将您使用的weibo数据集发到我的邮箱么？
758160601@qq.com或13918523648@163.com@zxh20001117
